libs/analytics/src/googleAnalytics/initGoogleAnalytics.ts

```diff
@@ -0,0 +1,38 @@
+import { CowAnalyticsGoogle } from './CowAnalyticsGoogle'
+
+import { CowAnalytics } from '../CowAnalytics'
+
+export const GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY = 'ga_client_id'
+
+export function initCowAnalyticsGoogle(): CowAnalytics {
+  // Load stored client id
+  let storedClientId: string | null = null
+  if (typeof window !== 'undefined') {
+    storedClientId = window.localStorage.getItem(GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY)
+  }
```

Comment:
Newbie question: does this mean that SSR won't have client ids?
Of course not, but the client side won't ever be hydrated with this sort of info?

Reply:
I think it will, at least this is what's happening right now. I saw it was working with this approach. It won't try to persist anything in the local storage for SSR rendering but it will for client rendering. I didn't get too deep on why it was working... as it just works :)